client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -99,11 +96,17 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         changeUnassginedUserFilter, unassignedUserFilter, changeInactiveUserFilter, inactiveUserFilter, fetchAllCountyUsers,
         tableTitle, timeRangeFilter, isBadgeInVisible, changeTimeRangeFilter
     } = useInvestigationTable({
-        setSelectedRow, allGroupedInvestigations, setAllCounties, setAllStatuses,setAllSubStatuses, setAllDesks, currentPage, setCurrentPage, setAllGroupedInvestigations,
-        investigationColor
+        setSelectedRow, allGroupedInvestigations, setAllStatuses, currentPage, setCurrentPage, setAllGroupedInvestigations,
+        investigationColor, setAllSubStatuses
     });
 
     const user = useSelector<StoreStateType, User>(state => state.user.data);
+    const displayedCounty = useSelector<StoreStateType, number>(state => state.user.displayedCounty);
+    const desks = useSelector<StoreStateType, Desk[]>(state => state.desk).filter(desk => desk.county === displayedCounty);
+
+    const countyDesks : Desk[] = useMemo(() =>
+        [...desks.filter(desk => desk.county === displayedCounty), { id: null, deskName: 'לא שוייך לדסק', county: displayedCounty}]
+    , [desks, displayedCounty])
```

Comment:
Why memo and not useEffect (with countydesks as a state or something)?

Reply:
because I figured that useMemo's role is just that - to create a mutated variable, so I dont see why using useEffect